src/System.Private.Xml/src/System.Private.Xml.csproj

```diff
@@ -7,7 +7,9 @@
     <RootNamespace>System.Xml</RootNamespace>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <NoWarn>$(NoWarn),649</NoWarn>
-    <DefineConstants Condition="'$(TargetGroup)' == 'uap'">$(DefineConstants);uap</DefineConstants>
+    <FeatureCompiledXsl Condition="'$(TargetGroup)' != 'uap' AND '$(TargetGroup)' != 'uapaot'">true</FeatureCompiledXsl>
+    <DefineConstants Condition="'$(FeatureCompiledXsl)' == 'true'">$(DefineConstants);FEATURE_COMPILED_XSL</DefineConstants>
+    <DefineConstants Condition="'$(TargetGroup)' == 'uapaot'">$(DefineConstants);FEATURE_SERIALIZATION_UAPAOT</DefineConstants>
```

Comment:
I think that it is confusing. There is nothing fundamentally AOT about it. The `FEATURE_` names should not be capturing the policy (which serializers are compiled into UAP or UAPAOT configs).

Take a look at how it is done in https://github.com/dotnet/corefx/blob/master/src/System.Linq.Expressions/src/System.Linq.Expressions.csproj#L13. This one should be similar. Also, notice the policy - we chose to compile the Linq expression interpreter into the CoreCLR builds, even though it is not strictly necessary for functional correctness (it is compiled in to give people choice about the perf characteristics).